docs/essentials/launcher.md

```diff
@@ -35,6 +35,27 @@ public class LauncherTest
 }
 ```
 
+## Platform Differences
+
+# [Android](#tab/android)
+
+Calling the `CanOpenAsync` method does not add any delay as it returns immediately. The `Result` of that Task is available immediately after the call.
+
+# [iOS](#tab/ios)
+
+Calling the `CanOpenAsync` method does not add any delay as it returns immediately. The `Result` of that Task is available immediately after the call.
+
+If the destination application on this device has never been opened by your application before, iOS will prompt the user once to allow your app to open it.
+
+More information about this function is available on the [iOS Documentation](https://developer.apple.com/documentation/uikit/uiapplication/1622952-canopenurl)
```

Comment:
Reword, since the information is not exactly 'about this function' but rather the underlying platform implementation,  and also please reference our API documentation rather than linking to apple
https://developer.xamarin.com/api/member/UIKit.UIApplication.CanOpenUrl/p/Foundation.NSUrl/